editor-style-rtl.css

```diff
@@ -1882,7 +1882,7 @@ input[type="submit"] {
   input[type="button"]:hover,
   input[type="reset"]:hover,
   input[type="submit"]:hover {
-    background: transparent !important; }
+    background: transparent; }
```

Comment:
@EvanHerman This is causes a bug with all submit buttons on the site, regardless of if WooCommerce is in use.

<img width="298" alt="screen shot 2017-02-15 at 2 15 23 pm" src="https://cloud.githubusercontent.com/assets/522158/22993630/3e7bb918-f389-11e6-813b-46aace8d8551.png">